src/shipit_workflow/shipit_workflow/api.py

```diff
@@ -20,10 +21,23 @@
 from shipit_workflow.models import Phase
 from shipit_workflow.models import Release
 from shipit_workflow.tasks import UnsupportedFlavor
+from shipit_workflow.tasks import fetch_actions_json
+from shipit_workflow.tasks import generate_action_task
 
 log = get_logger(__name__)
 
 
+def _queue():
+    queue = taskcluster.Queue({
```

Comment:
We may want to use `taskcluster.async.Queue` at some point! at which point we could use `async with taskcluster.async.Queue(...) as queue: yield_ queue`. This means the workflow won't hang on task creation, but would require asyncio.

Reply:
Good idea! We already use python 3.5, so it's worth to try it. Adding to TODO.